src/index.html

```diff
@@ -20,8 +21,424 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="#MoyoLogo"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
```

Comment:
pls format your html code according to this rule from style guide
![image](https://github.com/mate-academy/layout_catalog/assets/108538921/3f27fd8f-d352-41e5-8a12-5cb41057fcfe)